slackbridge/factories.py

```diff
@@ -57,6 +57,7 @@ def buildProtocol(self, addr):
 
     def add_user_bot(self, user_bot):
         IRCBot.users[user_bot.user_id] = user_bot
+        IRCBot.slack_nicks.append(user_bot.nickname.split('-')[0])
```

Comment:
I'm not sure if this is a valid concern, but considering that some users have "creative" slack usernames, perhaps it would be safer to just strip off the "-slack" part of the nick rather than splitting and selecting the first part? i.e. if someone decides to name themselves "break-this" on slack or something like that?

Reply:
Ergh... Good point. Even then, someone could still make their display name ```keur-slack-break``` and tag me. I think using the slack display names retrieved from the API is our best bet instead of splitting or stripping. 

I have to get back to my day job but I'll be sure to fix this!